lib/test.dart

```diff
@@ -8,6 +8,7 @@ import 'package:path/path.dart' as p;
 
 import 'src/backend/declarer.dart';
 import 'src/backend/invoker.dart';
+import 'src/backend/platform_selector.dart' show PlatformSelector; // for the dartdocs
```

Comment:
Apparently it only caused a dartfmt error.

In any case, I'm not sure this is the right fix; @jcollins-g may have better ideas. I just happened to notice that we're referring to `[PlatformSelector]` but dartdocs doesn't know why or what to link to.